docs/hub/datasets-pyarrow.md

```diff
@@ -0,0 +1,234 @@
+
+## Save a Table
+
+You can save a pyarrow Table using `pyarrow.parquet.write_table` to a local file or to Hugging Face directly.
+
+To save the Table on Hugging Face, you first need to [Login with your Hugging Face account](/docs/huggingface_hub/quick-start#login), for example using:
+
+```
+huggingface-cli login
+```
+
+Then you can [Create a dataset repository](/docs/huggingface_hub/quick-start#create-a-repository), for example using:
+
+```python
+from huggingface_hub import HfApi
+
+HfApi().create_repo(repo_id="username/my_dataset", repo_type="dataset")
+```
+
+Finally, you can use [Hugging Face paths](/docs/huggingface_hub/guides/hf_file_system#integrations) in PyArrow:
+
+```python
+import pyarrow.parquet as pq
+
+pq.write_table(table, "hf://datasets/username/my_dataset/imdb.parquet", use_content_defined_chunking=True)
+
+# or write in separate files if the dataset has train/validation/test splits
+pq.write_table(table_train, "hf://datasets/username/my_dataset/train.parquet", use_content_defined_chunking=True)
+pq.write_table(table_valid, "hf://datasets/username/my_dataset/validation.parquet", use_content_defined_chunking=True)
+pq.write_table(table_test , "hf://datasets/username/my_dataset/test.parquet", use_content_defined_chunking=True)
+```
+
+We use `use_content_defined_chunking=True` to enable faster uploads and downloads from Hugging Face thanks to Xet deduplication (it requires `pyarrow>=21.0`).
+
+<Tip>
+
+Content defined chunking (CDC) makes the Parquet writer chunk the data pages in a way that makes duplicate data chunked and compressed identically.
+Without CDC, the pages are arbitrarily chunked and therefore duplicate data are impossible to detect because of compression.
+Thanks to CDC, Parquet uploads and downloads from Hugging Face are faster, since duplicate data are uploaded or downloaded only once.
+
+</Tip>
+
+Find more information about Xet [here](https://huggingface.co/join/xet).
+
+## Use Images
+
+You can load a folder with a metadata file containing a field for the names or paths to the images, structured like this:
+
+```
+Example 1:            Example 2:
+folder/               folder/
+├── metadata.parquet  ├── metadata.parquet
+├── img000.png        └── images
+├── img001.png            ├── img000.png
+...                       ...
+└── imgNNN.png            └── imgNNN.png
+```
+
+You can iterate on the images paths like this:
+
+```python
+from pathlib import Path
+import pyarrow as pq
+
+folder_path = Path("path/to/folder")
+table = pq.read_table(folder_path + "metadata.parquet")
+for file_name in table["file_name"].to_pylist():
+    image_path = folder_path / file_name
+    ...
+```
+
+Since the dataset is in a [supported structure](https://huggingface.co/docs/hub/en/datasets-image#additional-columns) (a `metadata.parquet` file with a `file_name` field), you can save this dataset to Hugging Face and the Dataset Viewer shows both the metadata and images on Hugging Face.
```

Comment:
```suggestion
Since the dataset is in a [supported structure](https://huggingface.co/docs/hub/en/datasets-image#additional-columns) (a `metadata.parquet` file with a `file_name` field), you can save this dataset to Hugging Face and the Dataset Viewer shows both the metadata and images.
```
redundant? (not sure)